index.bs

```diff
@@ -1194,6 +1194,13 @@ a numbered step. If outdented, it (today) is rendered as a bullet in the midst o
                         :   {{AuthenticatorAttestationResponse/attestationObject}}
                         ::  |attestationObject|
 
+                        :   {{AuthenticatorAttestationResponse/transports}}
+                        ::  A sequence of one or more {{AuthenticatorTransport}} values. The first value is set to the transport used to obtain this registration. Further values are appended to indicate other, distinct, transports that the |authenticator| is believed to support.
+
+                            If a user agent does not wish to divulge this information it MAY choose not to include additional transports and MAY choose the first element to be a generic value rather than the actual transport used. But it MUST still include at least one value and it takes the risk that [=RP=] behaviour may be suboptimital.
```

Comment:
> generic value

Which Generic Value??

Reply:
See also the second part of my comment [here](https://github.com/w3c/webauthn/pull/1050#issuecomment-420714358) but I'm deliberately not specifying the value. The idea is that browsers may fabricate a plausible list and the list with the largest anonymity set may change over time. Thus the best choice of fabrication may also change over time so I don't want to nail it down in the spec.

However, this is evidently a point where I've been unclear and I shall change the wording here to improve.

Reply:
The term "generic value" has been dropped.